test/modules/http/compliance/http-req-method.test.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @vitest-environment node
+ */
+import { it, expect, beforeAll, afterAll } from 'vitest'
+import http from 'node:http'
+import { ClientRequestInterceptor } from '../../../../src/interceptors/ClientRequest'
+import { waitForClientRequest } from '../../../helpers'
+
+const interceptor = new ClientRequestInterceptor()
+
+beforeAll(() => {
+  interceptor.apply()
+})
+
+afterAll(() => {
+  interceptor.dispose()
+})
+
+it('Lowercased get method should work', async () => {
+  const request = http.request('http://example.com', { method: 'get' }).end()
```

Comment:
Could you please provide a mocked response via the interceptor? Right now, we are hitting actual example.com in this test. Thanks!